src/lib/compile-nools-rules.js

```diff
@@ -17,17 +15,31 @@ function lint(code) {
 function compileWithDefaultLayout(projectDir) {
   checkForRequiredFilesForDefaultLayout(projectDir);
 
-  const targets = parseTargets.js(projectDir);
+  const targets = fs.read(`${projectDir}/targets.js`);
   const tasks = fs.read(`${projectDir}/tasks.js`);
   const supportCode = fs.read(`${projectDir}/nools-extras.js`);
   const noolsLib = fs.read(`${__dirname}/../nools/lib.js`);
 
   const jsCode = templatedJs.fromString(projectDir, `
     var idx1, idx2, r, target;
     var now = Utils.now();
-    ${supportCode}
-    var targets = ${jsToString(targets)};
-    var tasks = ${tasks};
+    var extras = (function() {
+      var module = {};
```

Comment:
I updated standard config to run against this code (module.export-ed all `nools-extras` functions and variables + module.export-ed the arrays in `tasks` and `targets`) and this is where I hit an error in the app:
```
Error getting tasks TypeError: Cannot set property 'MS_IN_DAY' of undefined
    at eval (eval at parseAction (index.js:58), <anonymous>:5:349)
```
When this code is minified below, `module` becomes `i` and `exports` is undefined. I hotpatched it by editing this line to:
```
var module = { exports: {} };
```

Were you thinking of a different way of updating the config where this doesn't happen? I just did what seemed most obvious to me.

Reply:
Interesting.. I'm using es6 for muso and as such, have to disable the minify step. I'll do another test pass against standard now. Your fix seems reasonable.

Reply:
Ya. Good test. Embarrassing to not have tested the mainline scenario. :P